source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveActionRenderer.cs

```diff
@@ -71,20 +71,17 @@ public static Button CreateActionButton(AdaptiveAction action, AdaptiveRenderCon
                 Style = context.GetStyle($"Adaptive.{action.Type}"),
             };
 
-            if(action.Sentiment == AdaptiveSentiment.Positive || action.Sentiment == AdaptiveSentiment.Destructive)
+            if(!String.IsNullOrWhiteSpace(action.Sentiment))
             {
                 Style sentimentStyle = context.GetStyle($"Adaptive.{action.Type}.{action.Sentiment}");
 
-                if (sentimentStyle == null)
+                if(sentimentStyle == null && String.Equals(action.Sentiment, "positive", StringComparison.OrdinalIgnoreCase))
```

Comment:
>f( [](start = 17, length = 2)

looks like the style in the rest of this file places a space between the `if` and the conditional